### PR TITLE
Update PPP, Add Sample Consultation Data, Add Consultation Tests

### PR DESCRIPTION
**Pull request by juslam19**
- Update Justin Lam PPP
- Add Sample Consultation Data
- Add Consultation Test Cases
- Standardised Date Format displayed in cards

### CODE REVIEW

File: docs/team/juslam19.md [hunk 1]
```diff
@@ -5,23 +5,61 @@ title: Justin Lam's Project Portfolio Page
...
+  * Modified others' sections if there is incorrect documentation. Pull request: [#226](https://github.com/AY2122S2-CS2103T-T11-1/tp/pull/226)
+
+* **Contributions to the DG:**
+  * Added description of how consultation commands works, and UML diagram of the Consultation command logic. Pull request: [#90](https://github.com/AY2122S2-CS2103T-T11-1/tp/pull/90)
 
-* **Enhancements to existing features**: to be added soon
+* **Contributions to Team-Based Tasks:**
+  * Contributed to product concept brainstorming.
+  * Contributed to brainstorming of user stories.
+  * Contributed to adapting of AB3 UG to the MedBook product (no traces of AB3 left).
```
Comment (joey-chance) [inline docs/team/juslam19.md:53]: Is there really no traces of AB3 left? Are there no commands from AB3 that we still have? 🤔

File: docs/team/juslam19.md [hunk 2]
```diff
@@ -5,23 +5,61 @@ title: Justin Lam's Project Portfolio Page
...
+  * Contributed to the updating of AB3 DG sections (product scope, user stories, use cases, NFRs, etc.) to the MedBook product.
+  * Attended weekly meetings, implemented teammates' comments.
```
Comment (joey-chance) [inline docs/team/juslam19.md:55]: Would appreciate it if you were transparent with your absence most of the time too.

**Review by joey-chance — CHANGES_REQUESTED** [review]

File: docs/team/juslam19.md (hunk 2 above)
Comment (juslam19) [reply docs/team/juslam19.md:55]: It was some of the time

**Review by joey-chance — COMMENTED** [review]
Looks good to me, let's wait for the others to review this too. @cheekean5848 @clement0010

**Review by joey-chance — APPROVED** [review]
LGTM! 👍

**Pull request merged**